core/src/TopNavDropDown.html

```diff
@@ -119,6 +120,9 @@ <h5 class="fd-action-bar__title fd-has-type-1">{node.label}</h5>
     helpers: {
       hasOpenUIicon(node) {
         return NavigationHelpers.isOpenUIiconName(node.icon);
+      },
+      getNodeLabel(node) {
```

Comment:
I've noticed we define the same function in 2 more components. Maybe it makes sense to find the central place for it to avoid duplicates.

Reply:
First issue (click twice will fixed in a different issue)

Reply:
second issue: imo it is fine, because we already have centralized `NavigationHelpers.isOpenUIiconName(node.icon)` and hasOpenUIicon is fine... we need it as a **delegate** function.

Reply:
ok, thanks for the explanation 👍